$OpenBSD: patch-get_iplayer,v 1.2 2009/04/19 13:32:20 sthen Exp $

- use id3tag (from audio/id3lib) by default
- workaround requirement for newer p5-libwww

--- get_iplayer.orig	Sun Apr 19 10:41:29 2009
+++ get_iplayer	Sun Apr 19 14:26:14 2009
@@ -455,7 +455,7 @@ sub find_matches {
 	$lame_opts		.= ' --quiet ' if $opt->{quiet};
 	$vlc			= $opt->{vlc} || 'cvlc';
 	$vlc_opts		= '-vv' if $opt->{verbose} || $opt->{debug};
-	$id3v2			= $opt->{id3v2} || 'id3v2';
+	$id3v2			= $opt->{id3v2} || 'id3tag';
 	$tee			= 'tee';
 	$rtmpdump		= $opt->{rtmpdump} || 'rtmpdump';
 	$bandwidth		= $opt->{bandwidth} || 512000; # Download bandwidth bps used for rtsp streams
@@ -1187,7 +1187,7 @@ sub download_block {
 sub create_ua {
 	my $agent = shift;
 	my $ua = LWP::UserAgent->new;
-	$ua->timeout([$lwp_request_timeout]);
+	#$ua->timeout([$lwp_request_timeout]);
 	$ua->proxy( ['http'] => $opt->{proxy} ) if $opt->{proxy};
 	$ua->agent( $user_agent{$agent} );
 	$ua->conn_cache(LWP::ConnCache->new());
